atlasaction/action_test.go

```diff
@@ -355,20 +355,20 @@ func TestMigrateLint(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, 0, len(out))
 	})
-	t.Run("lint with error", func(t *testing.T) {
```

Comment:
this test is included in the test case bellow: `lint summary - lint error`, thus I deleted it